### ISSUE TITLE
`a` is not not capitalized in "Ads received this month" string

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#42697]
The first `a` should be capitalized. 

![Image](https://github.com/user-attachments/assets/18c3a77c-5a67-4b57-81ee-cac2e2ff8ca5)

cc: @LaurenWags @GeetaSarvadnya

**Comment by GeetaSarvadnya** [comment]
Verification INPROGRESS on

```
Brave | 1.75.120 Chromium: 132.0.6834.57 (Official Build) nightly (64-bit)
-- | --
Revision | 40377a8240bf613aa4c4cf7bce6283e73c2d1a48
OS | Windows 10 Version 22H2 (Build 19045.5247)
```

**Comment by Miyayes** [comment]
Looks like `a` is not capitalized in the non-connected state case: ![Image](https://github.com/user-attachments/assets/1d24625e-4a86-4530-af99-25fb7ccec9cf)
 @GeetaSarvadnya

**Comment by LaurenWags** [comment]
@Miyayes we probably need a separate issue for this since it's not the same exact string as mentioned when this was initially logged.

**Comment by Miyayes** [comment]
Yup, filed! Interestingly, the "Ads" in the connected state is capitalized properly (i.e., the Ads in the same position as the one in the non-connected state)!

**Comment by LaurenWags** [comment]
thanks for logging the follow up issue @Miyayes 😄
